### PR TITLE
Spree 3.0 support

### PR DESCRIPTION
**Pull request by damianlegawiec**
* Spree 3.0 support
* bootstrap frontend views
* bootstrap admin views

### CODE REVIEW

**Comment by ngrices** [comment]
@killy This is great! Do you think it is prudent to branch a 3.0 version at this point? or will master stay inline with Spree versions for a bit? I guess once we thought there was reason to invest in 3.X releases of spree that were not backwards compatible it would make sense to branch at that point.

**Comment by killy** [comment]
@ngrices most of the spree related gems use the branching for a specific version, so yeah 3-0-stable is a great idea. Maybe master should follow spree master? Currently, they are working on 3.1 which won't have many breaking changes to 3.0. Some serious fixes we can backport to the 2.x branches of the gem.

**Comment by killy** [comment]
@ngrices any news on this subject? :)

**Comment by ngrices** [comment]
Hi Damian
I will take a look at this tomorrow. Apologies for the delay as I have been out of pocket of the last week.
Thanks

Nathan Grice
Amazon Payments
c. 253.509.8001

On Jul 8, 2015, at 3:28 AM, Damian Legawiec <notifications@github.com<mailto:notifications@github.com>> wrote:


@ngrices<https://github.com/ngrices> any news on this subject? :)

-

**Comment by dustMason** [comment]
Any updates here? I would love specify `3-0-stable` in my Gemfile. Thanks!

File: spree_amazon_payments.gemspec [hunk 1]
```diff
@@ -14,7 +14,9 @@ Gem::Specification.new do |s|
   s.version     = '2.3.4'
```
Comment (ngrices) [inline spree_amazon_payments.gemspec]: So if I wanted to merge this right into master, shouldnt we rename our version to 3.0.0?

**Pull request merged**